tests/e2e/vsan_stretched_cluster.go

```diff
@@ -74,6 +74,12 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 		defaultDatastore           *object.Datastore
 		isVsanHealthServiceStopped bool
 		nimbusGeneratedK8sVmPwd    string
+		accessMode                 v1.PersistentVolumeAccessMode
+	)
+	const (
```

Comment:
move this code to e2e-common.go, instead of writing it in multiple files and read this const from there

Reply:
We are not keeping v1.* params in e2e common file